### ISSUE TITLE
"sslv3 alert certificate unknown" with some servers that require a client cert

### ISSUE THREAD

**Issue by nabla-c0d3** [nabla-c0d3/sslyze#531]
```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.7/3.7.12/Frameworks/Python.framework/Versions/3.7/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/Users/nabla/Documents/tools/sslyze/sslyze/scanner/_mass_connectivity_tester.py", line 108, in run
    network_configuration=scan_request.network_configuration,
  File "/Users/nabla/Documents/tools/sslyze/sslyze/server_connectivity.py", line 129, in check_connectivity_to_server
    tls_version=tls_detection_result.tls_version_supported,
  File "/Users/nabla/Documents/tools/sslyze/sslyze/server_connectivity.py", line 412, in _detect_ecdh_support
    ssl_connection.connect(should_retry_connection=False)
  File "/Users/nabla/Documents/tools/sslyze/sslyze/connection_helpers/tls_connection.py", line 292, in connect
    self.ssl_client.do_handshake()
  File "/Users/nabla/Documents/tools/sslyze/.venv/lib/python3.7/site-packages/nassl/ssl_client.py", line 182, in do_handshake
    self._ssl.do_handshake()
nassl._nassl.OpenSSLError: error:14094416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown
```

**Comment by nabla-c0d3** [comment]
Fixed by https://github.com/nabla-c0d3/nassl/commit/69a788490497c133b46dd66d8c7263084e163708 . Will need a new release of nassl.

**Comment by nabla-c0d3** [comment]
Fixed in [v5.0.0](https://github.com/nabla-c0d3/sslyze/releases/tag/5.0.0).
